### PR TITLE
Update version control guide with MxToolSet

### PR DESCRIPTION
**Pull request by Abulero**
New MxToolSet Merge feature allows the user to merge projects using git on a command line

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@Abulero , could you please sign CLA?

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md [hunk 1]
```diff
@@ -297,6 +297,39 @@ The second method should be used if the first method is not possible for some re
 3. Commit your changes using Studio Pro. 
 4. Reopen the main line app in Studio Pro only after overwriting the files.
 
+#### 7.2.5 Merging Using Git in the Command Line
+
+For merging MPRs using Git in the command line to work, it is necessary to attach mx.exe merge feature to Git as a driver.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md]: What is MPR?

Comment (Abulero) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md]: It's the .mpr file that contains all of the app's modules, documents etc

**Review by MariaShaposhnikova — APPROVED** [review]

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md [hunk 2]
```diff
@@ -297,6 +297,39 @@ The second method should be used if the first method is not possible for some re
 3. Commit your changes using Studio Pro. 
 4. Reopen the main line app in Studio Pro only after overwriting the files.
 
+#### 7.2.5 Merging Using Git in the Command Line
+
+For merging *.mpr* files using Git in the command line to work, it is necessary to attach mx.exe merge to Git as a driver.
+
+When doing a **git merge** operation on two branches in the command line, Git attempts to merge the binaries of *.mpr* files, which does not work. You need to apply Studio Pro merge algorithm and that is where mx.exe as a driver is needed.
+
+Navigate to the *.gitconfig* file in C:/Users/[USER_NAME] and add the following:
```
Comment (rik-bos) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md:306]: Could this also be configured in the local / repository config `.git/config` so the scenario in which a user has two separate project that each work with different SP versions can be supported?

In that case let's add that as an example / alternative as well. E.b. by mentioning:

Navigate to the git config file (for all project repositories of a user, the *.gitconfig* file in C:/Users/[USER_NAME], or specific per project repository `[project_dir]/.git/config` and add the following:

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md:306]: @Abulero , could you please take a look at Rik's question?

Comment (Abulero) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro.md:306]: Yes. It is definitelly possible to configure a git driver for a specific repository. There's a section at the bottom that explains the easiest way of doing so. I'll bring that up to this section !

**Pull request merged**